packages/suite/src/utils/suite/transactionReview.ts

```diff
@@ -0,0 +1,23 @@
+import { TranslationKey } from '@suite-common/intl-types';
+import { FormState } from '@suite-common/wallet-types';
+
+export const getTitleOrBtnText = (
+    ethereumStakeType: FormState['ethereumStakeType'],
+    isRbfAction: boolean,
+): TranslationKey => {
```

Comment:
```suggestion
export const getTitleOrBtnText = ({
    ethereumStakeType,
    isRbfAction
}): TypeHere => {
```
I would use destructured parameters in this case

Reply:
This is a bit confusing, because you’re suggesting to change the return type for the function. Don’t get how it helps with destructuring. But I think I’ve got your point.

https://github.com/everstake/trezor-suite/pull/4/commits/f258205008a78970a6f132c5d995383205820251